<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopulationPlot</class>
 <widget class="QWidget" name="PopulationPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Population plot</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QSpinBox" name="columns">
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="bins">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QSlider" name="slider">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="pageStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>500</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>bins</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>columns</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QComboBox" name="fits"/>
   </item>
   <item row="0" column="5">
    <widget class="QComboBox" name="mode">
     <item>
      <property name="text">
       <string>Unweighted histogram</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Weight histogram by cost</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Weight histogram by validation error</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Inverse mean cost in bin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Inverse min cost in bin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Inverse mean validation error in bin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Inverse min validation error in bin</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QComboBox" name="validations"/>
   </item>
   <item row="2" column="1" colspan="7">
    <widget class="QScrollArea" name="scrollArea">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QCustomPlot" name="panel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>923</width>
        <height>573</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QCheckBox" name="bestCost">
     <property name="text">
      <string>Best (blue)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QCheckBox" name="bestVali">
     <property name="text">
      <string>Best (green)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QCheckBox" name="target">
     <property name="text">
      <string>Target (pink)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QPushButton" name="pdf">
     <property name="text">
      <string>PDF...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
